server/controllers/usersign.js

```diff
@@ -0,0 +1,108 @@
+import bcrypt from 'bcrypt';
+import '@babel/plugin-transform-regenerator';
+import '@babel/polyfill';
```

Comment:
What do you use babel-polyfill for? I guess it serves no purpose here

Reply:
This helps me  to use the full set of ES6 features beyond syntax changes.

Reply:
But here I imported it expecting to use it. But It's not necessary.